WcaOnRails/app/webpacker/lib/hooks/useLoadedData.js

```diff
@@ -35,50 +33,4 @@ const useLoadedData = (url) => {
   };
 };
 
-export const useManyLoadedData = (ids, urlFn) => {
```

Comment:
Why is this being deleted?

Reply:
I deleted it because it's no longer used. It was introduced in https://github.com/thewca/worldcubeassociation.org/pull/8422.

It was used in `UserSearch`, it was used to call multiple APIs in parallel. In the new flow, that API call is not needed as per my reply to other comment, where I've explained the old flow and new flow.